# Goal: Разница между желаемыми и текущими параметрами робота
float32 delta_distance
float32 delta_body_angular_velocity
float32 delta_plane_angular_velocity
float32 delta_linear_velocity
float32 delta_body_angle
float32 delta_plane_angle

---

# Feedback: Пустой блок, если feedback не используется
# (ROS2 требует секцию Feedback, даже если она не используется)

---

# Result: Подтверждение выполнения
bool success
